packages/grafana-ui/src/components/FileDropzone/FileListItem.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('The FileListItem component', () => {
   it('should show a progressbar when the progress prop has a value', () => {
     render(<FileListItem file={{ file: file({}), id: '1', error: null, progress: 6 }} />);
 
-    expect(screen.queryByText('Cancel')).not.toBeInTheDocument();
+    expect(screen.queryByText('Cancel upload')).not.toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByRole('button', { name: /cancel/i }))).not.toBeInTheDocument();
```
Even if it's `queryBy` query you can still use `*ByRole` here.

Reply:
actually this throws error because the element is not there. I check that it is not in the document here.

Reply:
Yeah, sorry, it was supposed to be `queryByRole` not `getByRole`.